hotel-service/src/main/java/edu/myhotel/hotel/service/services/RoomDetailInfo.java

```diff
@@ -0,0 +1,90 @@
+package edu.myhotel.hotel.service.services;
+
+import edu.myhotel.hotel.service.storage.RoomFacility;
+import edu.myhotel.hotel.service.storage.RoomType;
+
+import java.util.List;
+
+public class RoomDetailInfo {
```

Comment:
Please replace all setters and getters with lombok's annotations on a class: `@Getter` and `@Setter`.

*apply this comment to all models or DTO classes.